soc/riscv/riscv-ite/common/check_regs.c

```diff
@@ -147,6 +147,39 @@ IT8XXX2_REG_OFFSET_CHECK(adc_it8xxx2_regs, adc_vchs_ctrl[0].VCHCTL, 0x60);
 IT8XXX2_REG_OFFSET_CHECK(adc_it8xxx2_regs, adc_vchs_ctrl[2].VCHDATM, 0x67);
 IT8XXX2_REG_OFFSET_CHECK(adc_it8xxx2_regs, ADCDVSTS2, 0x6c);
 
+/* Watchdog register structure check */
+#ifdef CONFIG_SOC_IT8XXX2_REG_SET_V1
+IT8XXX2_REG_SIZE_CHECK(wdt_it8xxx2_regs, 0x62);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, ET1CNTOLR, 0x40);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, ET1CNTOHR, 0x41);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, ET2CNTOLR, 0x44);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, ET2CNTOHR, 0x45);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, ET2CNTOH2R, 0x46);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, EWDCNTOLR, 0x60);
+IT8XXX2_REG_OFFSET_CHECK(wdt_it8xxx2_regs, EWDCNTOHR, 0x61);
```

Comment:
It seems like the register space could be managed from the devicetree.

You should consider using multiple `reg` pairs to specify the base addresses for ETWCFG and ET1CNTOLR.

Reply:
Remove these unused watchdog registers. If these are used later, we will add them according to your method.